packages/expo-task-manager/CHANGELOG.md

```diff
@@ -8,6 +8,8 @@
 
 ### 🐛 Bug fixes
 
+- Fix removing tasks. ([#8348](https://github.com/expo/expo/pull/8612) by [@mczernek](https://github.com/mczernek))
```

Comment:
Isn't it better not to have such changelog entry at all than to have one that says nothing?